### ISSUE TITLE
ICINGA Notifications

### ISSUE THREAD

**Issue by ananthaa-advisory** [jjethwa/icinga2-slack-notification#8]
Greetings,

After adding the slack notifications in the host and service template, I am able to see only few alerts are being triggered to Slack channel. 
Do you know why it is.

```
template Host "generic-host" {
  max_check_attempts = 3
  check_interval = 1m
  retry_interval = 30s
  check_command = "hostalive"
  vars.notify_slack=true
}

/**
 * Provides default settings for services. By convention
 * all services should import this template.
 */
template Service "generic-service" {
  max_check_attempts = 5
  check_interval = 1m
  retry_interval = 30s
  vars.notify_slack=true
}

```

**Comment by jjethwa** [comment]
Hi @ananthaa-advisory 

This config along with the others from your previous issue https://github.com/jjethwa/icinga2-slack-notification/issues/7 look like they should work properly.  Can you check the Icinga2 log under /var/log/icinga2 to see if it's triggering the notification correctly?

**Comment by ananthaa-advisory** [comment]
hi @jjethwa ,

I checked and I am looking to for Disk critical alerts as I set them to Critical and I dont see them in /var/log/icinga2/icinga2.log file. However, I see alerts for other service checks.

But , I have notficed that when I send a custom notifications from ICINGA UI, the disk notifications is coming to Slack channel. I am not sure what I did wrong and where the fault is .

**Comment by ananthaa-advisory** [comment]
HI @jjethwa ,

I dont know what I did, i just commented out the parameter from templates.conf for vars.notify_slack=true under generic_host and generic_service templates and restarted icinga2 service and now the notifications are coming fine.

Checked the logs and it is showing up there as well. You are a great Guru.

By the way ,if you dont mind  can i have your mail address. 

Thanks for all your help sir :)

Aravind
